project/PACKAGERS.md

```diff
@@ -44,8 +44,7 @@ need to package Docker your way, without denaturing it in the process.
 To build Docker, you will need the following:
 
 * A recent version of Git and Mercurial
-* Go version 1.4 or later (Go version 1.5 or later required for hardware signing
-  support in Docker Content Trust)
+* Go version 1.6.2 or later
```

Comment:
LOL, teach me how to read!

Well, 1.6 may be sufficient, but bad because it has a known vulnerability; we don't test on anything below 1.6.2, so 😇, but I'm happy removing the `.2` if you think it's better